core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.html

```diff
@@ -0,0 +1,161 @@
+<div class="oppia-edit-thumbnail-container">
+  <div class="modal-header">
+    <h3>Upload Thumbnail</h3>
+  </div>
+
+  <div class="modal-body oppia-thumbnail-uploader-container">
+    <div class="oppia-thumbnail-uploader">
+      <div [hidden]="uploadedImage">
+        <strong>Please upload an SVG image.</strong>
+        <div class="alert alert-warning">
+          <span>Here are some guidelines to follow:</span>
+          <ul>
+            <li>Uploaded image should have an aspect ratio of {{ aspectRatio }}.</li>
+            <li>Uploaded image should have a transparent background.</li>
+            <li>Please ensure that the uploaded image is less than 100 KB.</li>
+          </ul>
+        </div>
+        <oppia-image-uploader (fileChanged)="onFileChanged($event)"
+                              [allowedImageFormats]="allowedImageFormats">
+        </oppia-image-uploader>
+      </div>
+      <div class="oppia-form-error" *ngIf="invalidImageWarningIsShown">
+        Please upload a valid SVG image.
+      </div>
+      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.tags.length">
```

Comment:
why `?`

Reply:
passing `invalidTagsAndAttributes` as undefined here initially so *ngIf doesn't executes using  a `?`, after there is some File change and there is a wrong svg with some tags or attributes `invalidTagsAndAttributes` contains some value and *ngIf executes.